pkgs/os-specific/darwin/apple-source-releases/Libsystem/default.nix

```diff
@@ -63,21 +63,28 @@ appleDerivation' stdenv {
     cat <<EOF > $out/include/TargetConditionals.h
     #ifndef __TARGETCONDITIONALS__
     #define __TARGETCONDITIONALS__
-    #define TARGET_OS_MAC           1
-    #define TARGET_OS_OSX           1
-    #define TARGET_OS_WIN32         0
-    #define TARGET_OS_UNIX          0
-    #define TARGET_OS_EMBEDDED      0
-    #define TARGET_OS_IPHONE        0
-    #define TARGET_IPHONE_SIMULATOR 0
-    #define TARGET_OS_LINUX         0
+    #define TARGET_OS_MAC               1
```

Comment:
Shouldn't be needed for the initial bootstrap build yet, removing the stdenv rebuild.

Reply:
You're right, forgot to remove it when adding it in `make-bootstrap-tools.nix`. >.<

Reply:
I'm still building the intermediate bootstrap-tools to verify that the preFixup hook actually works as expected btw.

Reply:
Overridden installPhase did the trick.

Reply:
Should I go ahead and mark this ready for review?